lib/kitchen/directions/bake_index/v1.rb

```diff
@@ -137,7 +137,7 @@ def bake(book:)
 
     def add_term_to_index(term_element, page_title)
       group_by = term_element.text.strip[0]
-      group_by = I18n.t(:eob_index_symbols_group) unless group_by.match?(/\w/)
+      group_by = I18n.t(:eob_index_symbols_group) unless group_by.match?(/[a-zA-Z]/)
```

Comment:
I think this will need to handle words that start with accent marks too?  

Can you change line 139 to be:

```ruby
group_by = I18n.transliterate(term_element.text.strip[0])
```

to remove accent marks on the group by? (https://stackoverflow.com/a/15696883) And also add a spec that a word starting with an accent mark gets grouped appropriately? (e.g. "éblahblah")